extensions/wikia/EditTagging/EditTaggingHooks.class.php

```diff
@@ -69,12 +73,18 @@ public static function onArticleSaveComplete( &$article, &$user, $text, $summary
 
 		$request = RequestContext::getMain()->getRequest();
 		$revision_id = $revision->getId();
-		$rte_mode = $request->getVal( 'RTEMode', null );
 
-		self::tagRevisionIfCategoryEdit( $revision_id, $request );
-		self::tagRevisionIfSourceEdit( $revision_id, $request );
-		self::tagRevisionIfRTESourceEdit( $revision_id, $rte_mode );
-		self::tagRevisionIfRTEWysiwygEdit( $revision_id, $rte_mode );
+		// Iterate over all handlers until out of handlers or a handler returns true
+		// for this request
+		$handler_iterator = self::getHandlers()->getIterator();
+		while ( $handler_iterator->valid() ) {
+			$result = call_user_func( $handler_iterator->current(), $revision_id, $request );
+			if ( $result ) {
+				break;
+			} else {
```

Comment:
Yes. I had it here for readability, but it's clear what happens without it so I removed it. Thanks!